python/ray/rllib/agents/ppo/appo_policy.py

```diff
@@ -238,11 +270,44 @@ def make_time_major(*args, **kw):
             value_targets=make_time_major(
                 batch_tensors[Postprocessing.VALUE_TARGETS]),
             vf_loss_coeff=policy.config["vf_loss_coeff"],
-            entropy_coeff=policy.entropy_coeff,
-            clip_param=policy.config["clip_param"])
+            entropy_coeff=policy.config["entropy_coeff"],
+            clip_param=policy.config["clip_param"],
+            cur_kl_coeff=policy.kl_coeff,
+            use_kl_loss=policy.config["use_kl_loss"])
 
     return policy.loss.total_loss
 
+def stats(policy, batch_tensors):
+    values_batched = _make_time_major(
```

Comment:
what's this time_majoring thing? can you give me full context?

Reply:
Make_time_major was used in Impala for vtrace processing I believe. APPO also uses vtrace in one of its loss function.